src/acl/Asn.cc

```diff
@@ -555,15 +556,15 @@ ACLASN::empty () const
 }
 
 void
-ACLASN::parse()
+ACLASN::parse(Acl::ArgumentParser &parser)
 {
     CbDataList<int> **curlist = &data;
     CbDataList<int> **Tail;
     CbDataList<int> *q = nullptr;
     char *t = nullptr;
 
     for (Tail = curlist; *Tail; Tail = &((*Tail)->next));
-    while ((t = ConfigParser::strtokFile())) {
+    while ((t = parser.strtokFile())) {
```

Comment:
Please add a PR description bullet point to remind us to disclose our expectation (as a PR comment in an official PR) that the vast majority of these calls will be adjusted again (to drop "optional" prefix) when we start requiring values.

Reply:
Added.